packages/core/src/index.tsx

```diff
@@ -14,10 +14,11 @@ import { InternalLog } from './InternalLog';
 import { ProxyConfiguration, ProxyType } from './ProxyConfiguration';
 import { SdkVerbosity } from './SdkVerbosity';
 import { TrackingConsent } from './TrackingConsent';
-import { DdTrace, DdRum } from './foundation';
+import { DdTrace } from './foundation';
```

Comment:
we are going to move `DdTrace` to `/trace/DdTrace` at some point later, right?

Reply:
Yes that's the goal. 
It will either be when we implement the trace mapper, or when we end the refactoring of the root of the SDK